### ISSUE TITLE
hyperparameters 

### ISSUE THREAD

**Issue by rifaadev** [jsyoon0823/GAIN#32]
How do you choose the best hyperparameters for GAIN ?

**Comment by jsyoon0823** [comment]
You can do some random search or greedy search to find the best hyper-parameters in terms of accuracy.

**Comment by rifaadev** [comment]
@jsyoon0823 how you can use those methods without using model.fit in your code ?

**Comment by jsyoon0823** [comment]
I somehow manually did those hyper-parameter optimization.
